versions/latest/modules/en/pages/installation-and-upgrade/rollbacks.adoc

```diff
@@ -17,49 +19,9 @@ Follow the instructions from this page when:
 * If you need to restore Rancher to its previous state at the same Rancher version, see the xref:rancher-admin/back-up-restore-and-disaster-recovery/restore.adoc[restore documentation].
 ====
 
+== Rolling Back to Rancher v2.8.0+
 
```

Comment:
```suggestion
```

We shouldn't list any specific version (unless it's a special case) because the listed version will be EOL eventually